### DISCUSSION TITLE
changing Models directory

### DISCUSSION THREAD

**Discussion by wisdombox44** [lllyasviel/stable-diffusion-webui-forge#1168] [Q&A]
"Hi everyone,
I have all my models stored in ComfyUI. I'm currently testing the Forge web UI and I'm wondering how to direct it to use the same model directory. I want to use all my checkpoints, LoRAs, UNets, etc., without moving them from ComfyUI. I've checked the Forge config file but couldn't find a main models directory setting."

**Comment by SuperChevyChevy** [comment]
check the "web-user.bat" file, set your comfy path in there.

**Comment by danilomaiaweb** [comment]
> "Hi everyone, I have all my models stored in ComfyUI. I'm currently testing the Forge web UI and I'm wondering how to direct it to use the same model directory. I want to use all my checkpoints, LoRAs, UNets, etc., without moving them from ComfyUI. I've checked the Forge config file but couldn't find a main models directory setting."


In your webui-user.bat set the path to checkpoint as show below:

set COMMANDLINE_ARGS=  --ckpt-dir "F:\ModelsForge\Checkpoints" --lora-dir "F:\ModelsForge\Loras"
"F:\ModelsForge\" is my path with my checkpoints e lora change to your path

**Reply by wisdombox44** [reply]
Thank you, it works. I have been looking for 2 days, You are very helpfull :)
